### DISCUSSION TITLE
Resizing a region where {start: 0} doesn't work

### DISCUSSION THREAD

**Discussion by logesh2496** [katspaugh/wavesurfer.js#2996] [Q&A]
If you change the first region from the regions [example](https://wavesurfer-js.org/examples/#regions.js) to below

wsRegions.addRegion({
    **start: 0,**
    end: 2,
    content: 'Immovable',
    color: randomColor(),
  })
  
  The resizing of that region doesn't work. its not moving.

**Comment by katspaugh** [comment]
Good catch, sounds like a bug!

**Reply by katspaugh** [reply]
It should be fixed now.

**Reply by logesh2496** [reply]
that was quick!!!! thanks
